packages/api/src/repository.js

```diff
@@ -42,26 +42,42 @@ class Repository {
     return result;
   }
 
-  async mayAccess (platform, login) {
-    const collection = client.collection('permitted-users/' + platform + '/users').where('login', '==', login);
+  async storeTicket (ticketToPerform) {
+    return this.createDoc('tickets/' + ticketToPerform.state, {
```

Comment:
stylistic nit, I've been tending to use template strings these days:

```
`tickets/${ticketToPerform.state}`
```

not a huge difference just my personal preference.